app.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+let express = require('express');
+let path = require('path');
+let favicon = require('serve-favicon');
```

Comment:
I'd probably remove favicon and parsers for the time being and try to keep this PR as barebones as possible to fill the description. Personally though I don't mind if these hang about. 

Just having a simple express server that responds on / with "hello world" seems like enough for this PR imo. The parsers and favicon, static files, templates etc feels like a step too far? Even if it is quite useful and will be something we will need eventually.